File: ce/snippets/visualbasic/CRMV8/queryvb/vb/uselinqwithdotnetdataservicesde3.vb
' <snippetuselinqwithdotnetdataservicesde3>


 Dim orderedAccounts = From a In orgContext.CreateQuery("account") _
                       Order By a("name") _
                       Select a("name")
 Console.WriteLine("Display accounts ordered by name")
 Console.WriteLine("================================")
 For Each name In orderedAccounts
  Console.WriteLine(name)
 Next name

' </snippetuselinqwithdotnetdataservicesde3>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/query/cs/uselinqwithdotnetdataservicesde3.cs
// <snippetuselinqwithdotnetdataservicesde3>


 var orderedAccounts = from a in orgContext.CreateQuery("account")
                       orderby a["name"]
                       select a["name"];
 System.Console.WriteLine("Display accounts ordered by name");
 System.Console.WriteLine("================================");
 foreach (var name in orderedAccounts)
 {
  System.Console.WriteLine(name);
 }

// </snippetuselinqwithdotnetdataservicesde3>